fe/fe-core/src/main/java/com/starrocks/analysis/ModifyFrontendAddressClause.java

```diff
@@ -0,0 +1,44 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.analysis;
+
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+import com.starrocks.common.AnalysisException;
+import com.starrocks.common.Pair;
+import com.starrocks.ha.FrontendNodeType;
+import com.starrocks.system.SystemInfoService;
+
+public class ModifyFrontendAddressClause extends FrontendClause {
+
+    protected String discardedHostPort;
+    protected String newlyEffectiveHostPort;
+
+    protected Pair<String, Integer> discardedHostPortPair;
+    protected Pair<String, Integer> newlyEffectivePortPair;
+    
+    protected ModifyFrontendAddressClause(String hostPort, FrontendNodeType role) {
+        super(hostPort, role);
+    }
+
+    public ModifyFrontendAddressClause(String discardedHostPort, String newlyEffectiveHostPort) {
+        super(discardedHostPort, FrontendNodeType.UNKNOWN);
+        this.discardedHostPort = discardedHostPort;
+        this.newlyEffectiveHostPort = newlyEffectiveHostPort;
+    }
+
+    @Override
+    public void analyze(Analyzer analyzer) throws AnalysisException {
```

Comment:
If you support the syntax, the user may convert from FQDN to IP, you should reject it when analyzing.